Makefile

```diff
@@ -61,10 +61,15 @@ debug: base
 .PHONY: test
 test: lint
 	BCRYPT=no npx mocha --recursive --exit
+
 .PHONY: test-full
 test-full: lint
 	npx mocha --recursive --exit
 
+.PHONY: test-fast
+test-fast:
+	BCRYPT=no npx mocha --recursive --exit --invert="--fgrep @slow"
```

Comment:
The Mocha docs show:

```
-i, --invert  Inverts --grep and --fgrep matches                     [boolean]
```

Which makes me think it should be `--fgrep @slow --invert`. But if this is working, then maybe it doesn't matter.